java/code/webapp/WEB-INF/pages/help/chatindex.jsp

```diff
@@ -1,16 +0,0 @@
-<%@ taglib uri="http://struts.apache.org/tags-html" prefix="html" %>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
-<%@ taglib uri="http://rhn.redhat.com/rhn" prefix="rhn" %>
-<%@ taglib uri="http://rhn.redhat.com/tags/list" prefix="rl" %>
-<%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
-
-<html>
-<head>
-</head>
-<body>
-    <script type="text/javascript" src="/javascript/highlander.js?cb=${rhn:getConfig('web.version')}"></script>
```

Comment:
Can we remove the javascript file as well? `./web/html/javascript/highlander.js`

Reply:
The js file we can't remove it. It's being used in 4 more places

Reply:
ahh, ok